### DISCUSSION TITLE
Gaussian curvatures caculated using libigl and Meshlab are inconsistent

### DISCUSSION THREAD

**Discussion by keblicuey** [libigl/libigl#2345] [Q&A]
As described in the tutorial([](https://libigl.github.io/libigl-python-bindings/tut-chapter1/)), the Gussian curvature in igl is caculated based on  Discrete Differential-Geometry Operators for Triangulated 2-Manifolds(_Meyer, 2003_). Following the tutorial, I obtained the caculated results, the max k(discrete gussian curvature) value is 6.28, the min k value is -0.01. However, the results are different in Meshlab caculating with the same method, the max k value is 0.047, the min k value is -0.001.  I wonder why the difference is so significant. Thanks for answering.
![image](https://github.com/libigl/libigl/assets/81084513/aac32d39-c071-43ab-9e18-95103855ae62)
![image](https://github.com/libigl/libigl/assets/81084513/9b755590-2547-4d00-9a72-9fafd7d02b2a)

**Comment by keblicuey** [comment]
And the plot seems to be wrong with igl, the surface is visualize in one color. I'm confused about what's problem with it.

**Reply by alecjacobson** [reply]
Without the input mesh, I can't be totally sure, but my guess is that the coloring is just being blown out by the different choice of behavior for boundary vertices. If you scale the colormaps to the same range you might see more consistent values between the two.

**Comment by keblicuey** [comment]
Sorry, forget to upload the mesh file.  The k value of a half of verticles are 2*pi caculated with igl. 
[mesh.zip](https://github.com/libigl/libigl/files/14031643/mesh.zip)
